mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -257,6 +257,18 @@ func resourceBigtableGCPolicyRead(d *schema.ResourceData, meta interface{}) erro
 	for _, fi := range ti.FamilyInfos {
 		if fi.Name == columnFamily {
```

Comment:
NIT: There are several nested if statements here. Is it possible to organize this a bit differently?

Maybe something like:

```
if fi.Name != columnFamily {
    continue
}

/// etc etc
```